lib/school_of_fish.rb

```diff
@@ -1,19 +1,29 @@
-require 'gosu'
```

Comment:
We use `Gosu::Image.load_tiles` in this file so I recommend keeping this here for now